README.md

```diff
@@ -63,7 +63,7 @@ You can visit [doc page](https://hugegraph.apache.org/docs/quickstart/hugegraph-
 
 ## License
 
-HugeGraph is licensed under Apache 2.0 License.
+HugeGraph is licensed under [Apache 2.0 License](https://github.com/apache/incubator-hugegraph/blob/master/LICENSE).
```

Comment:
```suggestion
HugeGraph is licensed under [Apache 2.0 License](./LICENSE).
```

One problem that comes to mind is that **different versions/branch** of the `LICENSE` should be **kept relatively consistent**, rather than **fixed**, and it does seem that it would be better to use a **relative path** (so that the LICENSE file is also in the same version?)

Reply:
HugeGraph is licensed under [Apache 2.0 License](LICENSE).

Reply:
done